peermusic/desktop
# Feature/torrent

**Testing**
1. Take a public domain music file: http://www.pdsounds.org/audio/download/780/Yah+rly.mp3
2. Associate two instances as devices.
3. Add the file to one of them.
4. Refresh the other.
5. Closer the one holding the file.
6. Set the remaining to sharing level "everyone".
7. Reload the instance.
8. Open https://btorrent.xyz/
9. Click seed, top right. Add the file.
10. Click download in your remaining instance.

---

See: peermusic/user-stories/issues/58
